ops/terraform/env/mgmt/alarms.tf

```
@@ -106,3 +111,39 @@ resource "aws_cloudwatch_metric_alarm" "ec2_failing_instances" {
   alarm_actions = [aws_sns_topic.victor_ops_alert.arn]
   ok_actions    = [aws_sns_topic.victor_ops_ok.arn]
 }
+
+## BFD-3520
+resource "aws_cloudwatch_log_metric_filter" "ec2_init_fail_count" {
+  name           = local.init_fail_filter_name
+  pattern        = local.init_fail_pattern
+  log_group_name = local.log_groups.cloudinit_out
+
+  metric_transformation {
+    name          = local.init_fail_metric_name
+    namespace     = local.this_metric_namespace
+    value         = "1"
+    default_value = "0"
+  }
+}
+
+resource "aws_cloudwatch_metric_alarm" "ec2_init_fail" {
+  alarm_name          = local.init_fail_alarm_name
+  comparison_operator = "GreaterThanThreshold"
+  evaluation_periods  = "1"
+
+  metric_name = local.init_fail_filter_name
+  namespace   = local.this_metric_namespace
+  period      = 60
+
+  statistic = "Maximum"
```

nit: I _think_ if the statistic is `Maximum` we'll only ever get `1` or `0` as the Metric value for the Alarm. While the Alarm would still work just fine, the value that it reports as breaking the threshold wouldn't necessarily be representative of the _number_ of instances that failed during the evaluation period. Basically, with `Maximum`, the Alarm because a binary "yes, some instances are failing" or "no, no instances are failing".

Not a _big_ deal, because it's honestly quite unlikely we'll be seeing more than a single instance fail initialization within a minute period.

```suggestion
  statistic = "Sum"
```

---

applied as requested